aspnetcore/signalr/hub-filters.md

```diff
@@ -0,0 +1,196 @@
+---
+title: Use hub filters in ASP.NET Core SignalR
+author: brecon
+description: Learn how to use hub filters in ASP.NET Core SignalR.
+monikerRange: '>= aspnetcore-5.0'
+ms.author: brecon
+ms.custom: mvc
+ms.date: 05/22/2020
+no-loc: [Blazor, "Identity", "Let's Encrypt", Razor, SignalR]
+uid: signalr/hub-filters
+---
+
+# Use hub filters in ASP.NET Core SignalR
+
+Hub filters allow logic to run before and after hub methods are invoked by clients. This article provides guidance for writing and using hub filters.
```

Comment:
Adding a callout that this is a 5.0+ feature.

```suggestion
Hub filters:

* Are available in ASP.NET Core 5.0 or later.
* Allow logic to run before and after hub methods are invoked by clients.

This article provides guidance for writing and using hub filters.
```

Reply:
Do we normally do that? I thought that was the main purpose of the version filter drop down

Reply:
It's something I've started doing so that it's very clear to customers (and to my team, for that matter) when a feature was introduced. Without such verbiage, customers have to toggle between different versions of the doc to figure it out.